src/target/riscv/riscv.c

```diff
@@ -4499,55 +4509,47 @@ COMMAND_HANDLER(riscv_exec_progbuf)
 	return ERROR_OK;
 }
 
-COMMAND_HANDLER(riscv_set_enable_eq_match_trigger)
+COMMAND_HANDLER(riscv_set_enable_trigger_feature)
 {
 	struct target *target = get_current_target(CMD_CTX);
 	RISCV_INFO(r);
 
-	if (CMD_ARGC == 0) {
-		command_print(CMD, "equality match trigger enabled: %s", r->enable_equality_match_trigger ? "on" : "off");
-		return ERROR_OK;
-	} else if (CMD_ARGC == 1) {
-		COMMAND_PARSE_ON_OFF(CMD_ARGV[0], r->enable_equality_match_trigger);
-		return ERROR_OK;
-	}
-
-	LOG_ERROR("Command takes 0 or 1 parameters");
-	return ERROR_COMMAND_SYNTAX_ERROR;
-}
+	if (CMD_ARGC == 2) {
+		bool enable_for_wp = false;
```

Comment:
[nitpick] since the default is true, I would change the initialization to `true` also.

Reply:
Addressed.